apps/api/src/routes/api/v1/applications/handlers.ts

```diff
@@ -1292,6 +1293,45 @@ export async function saveSecret(request: FastifyRequest<SaveSecret>, reply: Fas
 		return errorHandler({ status, message });
 	}
 }
+export async function saveManySecrets(request: FastifyRequest<SaveManySecrets>, reply: FastifyReply) {
+    try {
+        const { id } = request.params
+        const { secrets, forceAlreadyExist = true } = request.body
+
+        if (forceAlreadyExist) {
+            await prisma.secret.deleteMany({ where: { applicationId: id, name: { in: secrets.map(secret => secret.name) } } });
+        }
+        
+        await prisma.secret.createMany({
+            data: secrets.map(secret => {
+                return {
+                    name: secret.name,
+                    value: encrypt(secret.value.trim()),
+                    isBuildSecret: secret.isBuildSecret,
+                    isPRMRSecret: false,
+                    application: { connect: { id } }
+                }
+            }),
+            skipDuplicates: true,
+        });
+        await prisma.secret.createMany({
+            data: secrets.map(secret => {
+                return {
+                    name: secret.name,
+                    value: encrypt(secret.value.trim()),
+                    isBuildSecret: secret.isBuildSecret,
+                    isPRMRSecret: true,
```

Comment:
What is the difference between `isPRMRSecret: true` and `isPRMRSecret: false`?

Reply:
Seems redunant to have two `.createMany` here

Reply:
On Application,  You have secrets of application, normal, and secrets of Review or PR MR, when you get a simple deploy of a application, is used `isPRMRSecret: false`, but when have a review deploy or deploy of a PR MR, and this not is production, is used `isPRMRSecret: true` .

On this case, de common when you create a env, set in the primary version on 2 options.

Is created 2 methods with createMany, to make it simple to understand.

@simonorzel26, I believe it is better not to use both, and specify in the payload sent if the secret is PRMR or Not, do you agree?

Reply:
Ok easy seems reasonable